Form-Controls/styles.css

```diff
@@ -0,0 +1,145 @@
+/* Custom properties */
+:root {
```

Comment:
Could you check your Lighthouse Accessibility score and find what is the problem? Based on the [criteria](https://github.com/CodeYourFuture/Module-User-Focused-Data/blob/main/Form-Controls/README.md) Lighthouse Accessibility score should be 100.
<img width="903" alt="Screenshot 2024-10-30 at 19 12 41" src="https://github.com/user-attachments/assets/94fcdd44-a639-44f1-87da-fdda3f7d4411">

Reply:
Accessibility score is 100 now. 
![image](https://github.com/user-attachments/assets/f3ddca3f-6ed9-497b-bb71-16d913b4b22a)